<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_menu_ui</class>
 <widget class="QWidget" name="Settings_menu_ui">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>GPU ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gpu_id_spinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Axon Shape</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="axon_shape_comboBox">
       <item>
        <property name="text">
         <string>circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ellipse</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="done_button">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
